### PR TITLE
Update Set-SPOTenant.md

### PR DESCRIPTION
**Pull request by SanjoyanM**

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -1937,6 +1938,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -LabelMismatchEmailHelpLink
+This parameter allows tenant admins to customize the “Help Link” in email with subject "Incompatible sensitivity label detected". When a sensitivity label mismatch occurs between the label on the document uploaded and the label on the site, SharePoint Online captures an audit record, and sends an Incompatible sensitivity label detected email notification to the person who uploaded the document and the site owner. The notification contains details of the document which caused the problem and the label assigned to the document and to the site. The comparison happens between the priority of these two labels.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1942]:
```suggestion
This parameter allows tenant admins to customize the “Help Link” in email with the subject "Incompatible sensitivity label detected." When a sensitivity label mismatch occurs between the label on the document uploaded and the label on the site, SharePoint Online captures an audit record and sends an Incompatible sensitivity label detected email notification to the person who uploaded the document and the site owner. The notification contains details of the document which caused the problem and the label assigned to the document and to the site. The comparison happens between the priority of these two labels. 
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 2]
```diff
@@ -1937,6 +1938,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -LabelMismatchEmailHelpLink
+This parameter allows tenant admins to customize the “Help Link” in email with subject "Incompatible sensitivity label detected". When a sensitivity label mismatch occurs between the label on the document uploaded and the label on the site, SharePoint Online captures an audit record, and sends an Incompatible sensitivity label detected email notification to the person who uploaded the document and the site owner. The notification contains details of the document which caused the problem and the label assigned to the document and to the site. The comparison happens between the priority of these two labels. 
+
+The value can any valid url.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1944]:
```suggestion
The value can any valid URL.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @SanjoyanM, kindly commit the minor suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@SanjoyanM

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @get-itips @tseward

**Review by dariomws — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 3]
```diff
@@ -1937,6 +1938,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -LabelMismatchEmailHelpLink
+This parameter allows tenant admins to customize the “Help Link” in email with the subject "Incompatible sensitivity label detected." When a sensitivity label mismatch occurs between the label on the document uploaded and the label on the site, SharePoint Online captures an audit record and sends an Incompatible sensitivity label detected email notification to the person who uploaded the document and the site owner. The notification contains details of the document which caused the problem and the label assigned to the document and to the site. The comparison happens between the priority of these two labels. 
+
+The value can any valid URL.
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1944]:
```suggestion
The value can be any valid URL.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 4]
```diff
@@ -1937,6 +1938,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -LabelMismatchEmailHelpLink
+This parameter allows tenant admins to customize the “Help Link” in email with the subject "Incompatible sensitivity label detected." When a sensitivity label mismatch occurs between the label on the document uploaded and the label on the site, SharePoint Online captures an audit record and sends an Incompatible sensitivity label detected email notification to the person who uploaded the document and the site owner. The notification contains details of the document which caused the problem and the label assigned to the document and to the site. The comparison happens between the priority of these two labels.
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1942]:
```suggestion
This parameter allows tenant admins to customize the "Help Link" in email with the subject "Incompatible sensitivity label detected." When a sensitivity label mismatch occurs between the label on the document uploaded and the label on the site, SharePoint Online captures an audit record and sends an Incompatible sensitivity label detected email notification to the person who uploaded the document and the site owner. The notification contains details of the document which caused the problem and the label assigned to the document and to the site. The comparison happens between the priority of these two labels. 
```

**Comment by get-itips** [comment]
I am ok after suggestions applied. Thanks

**Review by tseward — APPROVED** [review]

**Comment by tseward** [comment]
Also Ok after changes are applied.

**Comment by yogkumgit** [comment]
@SanjoyanM please implement the suggested changes. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**